examples/guardrails-evaluation/README.md

```diff
@@ -0,0 +1,163 @@
+
+### 📰 Harmful Content - Information Integrity (2 plugins)
+
+- **harmful:misinformation-disinformation** - False information
+- **harmful:specialized-advice** - Unqualified advice
+
+### ⚖️ Harmful Content - Legal & IP (2 plugins)
+
+- **harmful:copyright-violations** - Copyright infringement
+- **harmful:intellectual-property** - IP theft
+
+### 🛡️ System Security & Integrity (6 plugins)
+
+- **cyberseceval** - Meta's cybersecurity dataset _(CRITICAL!)_
+- **excessive-agency** - Unauthorized actions _(CRITICAL!)_
+- **hallucination** - False information generation _(CRITICAL!)_
+- **overreliance** - Over-dependence issues _(CRITICAL!)_
+- **divergent-repetition** - Resource exhaustion attacks
+- **reasoning-dos** - Reasoning DoS attacks
+
+### 🔐 PII & Privacy Protection (3 plugins)
+
+- **pii:direct** - Direct PII disclosure
+- **pii:social** - Social engineering PII extraction
+- **harmful:privacy** - Privacy violations
+
+### ⚖️ Bias & Fairness (1 plugin)
+
+- **bias:gender** - Gender bias testing _(CRITICAL!)_
+
+### 🏢 Business & Compliance (4 plugins)
+
+- **contracts** - Unauthorized contractual commitments
+- **imitation** - Entity impersonation
+- **politics** - Political content/bias
+- **religion** - Religious content/bias
+
+## Usage
+
+### Basic Usage
+
+```yaml
+redteam:
+  plugins:
+    - guardrails-evaluation
+```
+
+### With Strategies (Enhanced Testing)
+
+```yaml
+redteam:
+  plugins:
+    - guardrails-evaluation
+  strategies:
+    - jailbreak
+    - prompt-injection
+    - multilingual
+```
+
+### Complete Example
+
+```yaml
+# yaml-language-server: $schema=https://promptfoo.dev/config-schema.json
+description: Comprehensive Guardrails Evaluation
+targets:
+  - openai:chat:gpt-4o-mini
+prompts:
+  - '{{prompt}}'
+redteam:
+  purpose: You are a helpful assistant
+  plugins:
+    - guardrails-evaluation
+  strategies:
+    - jailbreak
+    - prompt-injection
+    - multilingual
+defaultTest:
+  metadata:
+    purpose: You are a helpful assistant
+```
+
+## Success Criteria Alignment
+
+This preset is designed to test against the requirements specified in your POC document:
+
+- **✅ Prompt Injection Coverage**: 15+ injection techniques (80% coverage target)
+- **✅ MITRE Framework**: 9 attack categories (90% block rate target)
+- **✅ Harmful Content**: Comprehensive coverage of all major harm categories
+- **✅ PII Protection**: Multi-vector PII leakage testing
+- **✅ Bias Detection**: Gender bias and fairness testing
+- **✅ System Integrity**: Hallucination, overreliance, and security testing
+
+## Performance Expectations
+
+This comprehensive preset will generate significant test volume:
+
+- **51 plugins** × **5 tests/plugin** = ~255 test cases
+- **Estimated runtime**: 15-30 minutes depending on model speed
+- **Use `--max-concurrency` to optimize testing speed**
+
+## Related Files
+
+- `guardrails-redteam.yaml` - Example configuration
+- See [promptfoo documentation](https://promptfoo.dev/docs/red-team/) for more details
```

Comment:
consider adding a few more links here to guardrail assertions, how to evaluate guardrails guide, etc. 

https://www.promptfoo.dev/docs/guides/testing-guardrails/